cmd/agent/subcommands/run/command.go

```diff
@@ -333,6 +334,13 @@ func getSharedFxOption() fx.Option {
 		dogstatsd.Bundle(),
 		otelcol.Bundle(),
 		rcclient.Module(),
+		fx.Provide(func(config config.Component) tagger.Params {
+			if pkgconfig.IsCLCRunner() {
+				return tagger.NewCLCRunnerRemoteTaggerParams()
+			}
+			return tagger.NewTaggerParams()
+		}),
```

Comment:
since we deplicate the same 5 lines of code in several sun command can we create in the `tagger` Package a function

```
// package tagger
...
func NewTaggerParams (cfg config.Component) Params {
			if pkgconfig.IsCLCRunner(cfg) {
				return tagger.NewCLCRunnerRemoteTaggerParams()
			}
			return tagger.NewTaggerParams()
		}

```

Reply:
It would be great to have a named function to factorize this logic.
Note that other parts of the code use different logics so `NewTaggerParams` is too generic and a more  specific name should ideally be used.